packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -693,44 +693,43 @@ function wait(time) {
 }
 
 function handleErrorMsg(err) {
-  cliux.print(`Error: ${(err?.errorMessage || err?.message) ? err?.errorMessage || err?.message : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')}`, { color: 'red' })
+  cliux.print(
+    `Error: ${
+      err?.errorMessage || err?.message
+        ? err?.errorMessage || err?.message
+        : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')
+    }`,
+    { color: 'red' },
+  );
   process.exit(1);
 }
 
-async function apiRequestHandler(org, queryParam = {}) {
-  const headers = {
-    authtoken: configHandler.get('authtoken'),
-    organization_uid: org.uid,
-    'Content-Type': 'application/json',
-    api_version: 1.1,
-  };
-
-  return await new HttpClient()
-    .headers(headers)
-    .queryParams(queryParam)
-    .get(`${configHandler.get('region')?.cma}/organizations/${org?.uid}/teams`)
-    .then((res) => {
-      const { status, data } = res;
-      if (status === 200) {
-        return data;
-      } else {
-        cliux.print(`${data?.error_message || data?.message || data?.errorMessage}`, { color: 'red' });
-        process.exit(1);
-      }
+async function getAllTeamsUsingSDK(managementAPIClient, org, queryParam = {}) {
+  const teamsData = await managementAPIClient
+    .organization(org.uid)
+    .teams()
+    .fetchAll(queryParam)
+    .then((data) => {
+      return data;
     })
     .catch((error) => {
       handleErrorMsg(error);
     });
+  return teamsData;
 }
 
 async function exportOrgTeams(managementAPIClient, org) {
   let teamsObjectArray = [];
   let skip = 0;
   let limit = config?.limit || 100;
   do {
-    const data = await apiRequestHandler(org, { skip: skip, limit: limit, includeUserDetails: true });
+    const data = await getAllTeamsUsingSDK(managementAPIClient, org, {
+      skip: skip,
+      limit: limit,
+      includeUserDetails: true,
+    });
     skip += limit;
-    teamsObjectArray.push(...data?.teams);
+    teamsObjectArray.push(...data?.items);
```

Comment:
This line of code will throw an error if `data.item` can't present always present in the result, the spread operation will throw an error. If we are sure `data.item` will always be there we can remove the `Optional chaining (?.)` from `data?.item`

Reply:
Removed the optional chaining from this